src/displayBackend/drm/Dumb.cpp

```diff
@@ -145,8 +148,17 @@ void DumbDrm::copy()
 	}
 
 	DLOG(mLog, DEBUG) << "Copy dumb, handle: " << mBufDrmHandle;
-
-	memcpy(mBuffer, mGnttabBuffer->get(), mSize);
+	if(mStride == mFrontStride)
+	{
+		memcpy(mBuffer, mGnttabBuffer->get(), mSize);
+		return;
+	}
+	auto src = reinterpret_cast<unsigned char*>(mGnttabBuffer->get());
+	auto dst = reinterpret_cast<unsigned char*>(mBuffer);
+	for(unsigned int i = 0; i < mHeight; i++)
```

Comment:
use int

Reply:
mHeight is uint32_t, will be warning about a comparison between signed and unsigned
I think I should  use uint32_t

Reply:
makes sense, then just leave it as is